### PR TITLE
Update SemanticAttributes and ResourceAttributes to version 1.7.0

### LINKED ISSUE #196: Update Semantic Conventions to v1.3.0
https://github.com/open-telemetry/opentelemetry-swift/blob/main/Scripts/semantic-convention/generate.sh#L7

https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.3.0

We may want to wait for 1.4.0 before picking this up, as it includes mobile-specific conventions which is appropriate to this SIG package.

### PR DESCRIPTION
**Pull request by nachoBonafonte**
Automatically generated by the generator with a pair of compiler fixes

Closes #196

### CODE REVIEW

**Review by SergeyKanzhelev — APPROVED** [review]

File: Sources/OpenTelemetrySdk/Resources/ResourceAttributes.swift [hunk 1]
```diff
@@ -253,48 +247,117 @@ public enum ResourceAttributes: String {
...
+    // Examples
+    attributes[.deviceModelIdentifier] = "iPhone3,4"
+    attributes[.deviceModelIdentifier] = "SM-G920F"
+    ~~~
+
+    - Note: It's recommended this value represents a machine readable version of the model identifier rather than the market or consumer-friendly name of the device.
+
+    - Requires: Value type should be `String`
+    */
+    case deviceModelIdentifier = "device.model.identifier"
```
Comment (Sherlouk) [inline Sources/OpenTelemetrySdk/Resources/ResourceAttributes.swift:275]: These are the new attributes that we discussed in a SIG meeting a couple months back. Will want to add them (in a future PR) to the resource package.

**Comment by Sherlouk** [comment]
Awesome update - looks good. Just remember this is breaking change so will need to reflect that in the SemVer as such.

File: Sources/OpenTelemetrySdk/Resources/ResourceAttributes.swift (hunk 1 above)
Comment (bryce-b) [reply Sources/OpenTelemetrySdk/Resources/ResourceAttributes.swift:275]: I made an issue https://github.com/open-telemetry/opentelemetry-swift/issues/274

**Review by bryce-b — APPROVED** [review]

**Comment by nachoBonafonte** [comment]
> Awesome update - looks good. Just remember this is breaking change so will need to reflect that in the SemVer as such.

Yes, I was planning to release version 1.1.0 with these and other breaking changes

**Pull request merged**